cedar-policy-validator/src/cedar_schema/fmt.rs

```diff
@@ -32,7 +32,7 @@ impl<N: Display> Display for json_schema::Fragment<N> {
         for (ns, def) in &self.0 {
             match ns {
                 None => write!(f, "{def}")?,
-                Some(ns) => write!(f, "namespace {ns} {{\n{def}}}")?,
+                Some(ns) => write!(f, "namespace {ns} {{\n{def}}}\n")?,
```

Comment:
Fixes weird formatting noted in https://github.com/cedar-policy/cedar/issues/682#issuecomment-2059687349